build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id 'java-library'
+    id "com.diffplug.gradle.spotless" version "3.27.0"
+    id 'maven-publish'
+    id 'signing'
+    id 'io.codearte.nexus-staging' version '0.21.2'
+}
+
+group 'com.adscore'
+version '1.0.4' //version of samples match with the version of the client-libs-java project
+
+sourceCompatibility = 1.8
+ 
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.adscore:adscore-client-libraries:1.0.4'
```

Comment:
this also should be implementation (compile is deprecated)

Reply:
Fixed!

Reply:
Just figured out, what Gradle docs say:
`the implementation dependency configuration provides the same functionality as compile.` -> this is not true.

It makes difference when you build a fat jar (what we do here, to be allowed to just execute it and have all required dependencies inside). 

while using 'implementation' on a dependency, is not added to classpath while building a fat jar, therefore the sample is unexecutable :/

while using 'compile' here and then doing `gradle jar`, fat jar is packed properly, with com.github.seancfoley dependency inside it.
  
Base on this observation, I've reverted to using compile, because any way of building a fat jar by Gradle with 'implementation' dependency does not work. (checked with the latest version of Gradle, 6.8.3)

Reply:
okay, finally we've left `implementation`, and the fat jar properly (by collecting dependencies on `configurations.runtimeClasspath` not `configurations.compile`